packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -672,6 +677,32 @@ void main() {
         await controller.seekTo(const Duration(milliseconds: 300));
         expect(controller.value.caption.text, 'one');
       });
+
+      test('setClosedCapitonFile loads caption file', () async {
+        final VideoPlayerController controller = VideoPlayerController.network(
+          'https://127.0.0.1',
+        );
+
+        await controller.initialize();
+        await controller.setClosedCaptionFile(_loadClosedCaption());
+
+        await controller.seekTo(const Duration(milliseconds: 100));
+        expect(controller.value.caption.text, 'one');
+      });
+
+      test('setClosedCapitonFile removes/changes caption file', () async {
```

Comment:
Why is this called "removes/changes" if it's only testing removal?

Reply:
I thought removing actually is a changing. It is changing to null.